apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx

```diff
@@ -111,8 +111,8 @@ export default function ViewParamsEditor({ paramFields }: EditFormDrawerProps) {
         </DrawerBody>
 
         <DrawerFooter className={style.drawerFooter}>
-          <Button variant='ontime-ghosted' onClick={clearParams} type='reset'>
-            Clear
+          <Button variant='ontime-ghosted' onClick={resetParams} type='reset'>
+            Reset
```

Comment:
One for the future: I wish reset didnt close the drawer... what do you think?

Reply:
I can see two options:
- reset the values and start over
- reset the values and get to the original view

What do you think?

Reply:
I believe we would like to reset the options which would make the view re-render with default attributes 
This already happens, only difference is that we would leave the drawer open

Reply:
So would you like me to add the drawer to close?

Reply:
I didnt understand the question, but we can take this another time